core/domain/suggestion_registry.py

```diff
@@ -1481,6 +1481,205 @@ def to_dict(self):
+        domain object.
+
+        Returns:
+            dict. A dict representation of a QuestionContributionStats
+            domain object.
+        """
+        return {
+            'contributor_user_id': self.contributor_user_id,
+            'topic_id': self.topic_id,
+            'submitted_questions_count': self.submitted_questions_count,
+            'accepted_questions_count': (
+                self.accepted_questions_count),
+            'accepted_questions_without_reviewer_edits_count': (
+                self.accepted_questions_without_reviewer_edits_count),
+            'first_contribution_date': (
+                self.first_contribution_date),
+            'last_contribution_date': self.last_contribution_date
+        }
+
+
+class QuestionReviewStats:
+    """Domain object for the QuestionReviewStatsModel."""
+
+    def __init__(
+            self, contributor_user_id, topic_id,
+            reviewed_questions_count,
+            accepted_questions_count,
+            accepted_questions_with_reviewer_edits_count,
+            first_contribution_date, last_contribution_date):
+        self.contributor_user_id = contributor_user_id
+        self.topic_id = topic_id
+        self.reviewed_questions_count = reviewed_questions_count
+        self.accepted_questions_count = accepted_questions_count
+        self.accepted_questions_with_reviewer_edits_count = (
+            accepted_questions_with_reviewer_edits_count
+        )
+        self.first_contribution_date = first_contribution_date
+        self.last_contribution_date = last_contribution_date
+
+    def to_dict(self):
+        """Returns a dict representation of a QuestionContributionStats
+        domain object.
+
+        Returns:
+            dict. A dict representation of a QuestionContributionStats
+            domain object.
+        """
+        return {
+            'contributor_user_id': self.contributor_user_id,
+            'topic_id': self.topic_id,
+            'reviewed_questions_count': self.reviewed_questions_count,
+            'accepted_questions_count': (
+                self.accepted_questions_count),
+            'accepted_questions_with_reviewer_edits_count': (
+                self.accepted_questions_with_reviewer_edits_count),
+            'first_contribution_date': (
+                self.first_contribution_date),
+            'last_contribution_date': self.last_contribution_date
+        }
+
+
+class ContributorMilestoneEmailInfo:
+    """Stores key information that is used to create the email content for
+    notifying contributors about milestones they achieved.
+
+    Attributes:
+        contributor_user_id: str. The type of the suggestion.
+        language_code: str. The language code of the suggestion.
+        contribution_type: str. The type of the contribution i.e.
+            translation or question.
+        contribution_sub_type: str. The sub type of the contribution
+            i.e. submissions/acceptances/reviews/corrections.
```

Comment:
```suggestion
            i.e. submissions/acceptances/reviews/edits.
```

I don't think we use the "corrections" term anywhere.

Reply:
Done